.vscode/tasks.json

```diff
@@ -0,0 +1,27 @@
+// See https://go.microsoft.com/fwlink/?LinkId=733558
+// for the documentation about the tasks.json format
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "build",
+      "type": "npm",
+      "script": "dev:vscode",
+      "isBackground": true,
+      "presentation": {
+        "reveal": "never"
+      },
+      "problemMatcher": [
+        {
+          "base": "$ts-webpack-watch",
```

Comment:
Webpack isn't being used here so this doesn't appear to be the right problem matcher. I've switched it out for $tsc in https://github.com/fwouts/previewjs/commit/9919e32ca28c85147caf2bd5c60809051f1d4d6f, let me know what you think.